src/main/java/gregtech/api/unification/material/materials/UnknownCompositionMaterials.java

```diff
@@ -361,5 +361,53 @@ public static void register() {
 
         GreenSapphireSlurry = new Material.Builder(1622, "green_sapphire_slurry")
                 .fluid().color(0x64c882).build();
+
+        DyeBlack = new Material.Builder(1623, "dye_black")
+                .fluid().color(0x202020).build();
```

Comment:
Is there a place where we could get this color (all dyes anyway) from in vanilla? There might not be, but if there is, it would be good to use that instead

Reply:
It was decided that the current colors from GT5u are better than those of Vanilla MC.